mozci/console/application.py

```diff
@@ -4,12 +4,14 @@
 
 from cleo import Application
 
+from mozci.console.commands.decision import DecisionCommand
 from mozci.console.commands.push import PushCommands
 
 
 def cli():
     application = Application()
     application.add(PushCommands())
+    application.add(DecisionCommand())
```

Comment:
Maybe we want to name this command more explicitly? Something like `trigger-classifications`? Wdyt @ahal?

Reply:
Actually I kind of wonder if integrating into the cli at all is the right approach here, since no one will be able to run it anyway.. Maybe this should just be script and then the hook would explicitly run `python mozci/scripts/decision.py` or something like that?